Standards/scs-0101-v1-entropy.md

```diff
@@ -5,114 +5,127 @@ status: Draft
 track: IaaS
 ---
 
-## 1. Terminology of Entropy
+## Introduction
 
-### 1.1. Entropy in information Technology
+### Entropy in information technology
 
 Entropy is a concept that is widely used in the scope of information
 technology. It is a measurement of the amount of disorder or randomness in
-a system. Entropy is used to measure the amount of information in a self
-contained systems, as well as the amount of incertitude that exists in this
-system. For cryptographic procedures and operations good entropy
-is a must have!
+a system. Entropy is used to measure the amount of information in a
+self-contained system, as well as the amount of incertitude that exists in this
+system.
 
-In traditional baremetal systems the amount of incertitude is generated
-by the randomness of read/write cycle of the disk heads of a disk-drive,
-bus timings as well as items such as keyboard timings.
+### Real-world uses of entropy
 
-### 1.2 Entropy in Virtual Instances
+Cryptography is a very prominent, albeit not the only application that
+heavily relies on entropy for operations such as creating secure keys.
+When the available _entropy runs out_, said operations can stall and
+take an abnormally long amount of time, which in turn can lead to
+malfunctions, e.g., with OpenSSL or load balancers.
 
-Virtual instances or virtual machines do not have these sources
-for random numbers. An instance will operate normally, but as
-cryptographic operations happen, procedures will take an abnormal long time,
-because with a small entropy count cryptographic operation can not operate
-in realtime. Examples are malfunctioning applications and OpenSSL
-operations that will not work.
+### Sources of entropy
 
-```console
-  $cat /proc/sys/kernel/random/entropy_avail
-  128
-```
+In _traditional baremetal systems_ the amount of incertitude is sourced
+from the randomness of the read/write cycles of the disk heads of a disk drive,
+bus timings, or keyboard timings, to name a few.
 
-#### 1.2.1 How to generate entropy "Out-Of-Nothing" ?
+_More recent methods_ of generating entropy include measuring IRQ jitter
+(available in Linux since kernel 5.4 or, before that, via a daemon such as
+[HavegeD](http://www.issihosts.com/haveged/)) as well as the special CPU
+instructions RDRAND and RDSEED, which are present in all modern CPUs from
+ARM, AMD, and Intel.
```

Comment:
ARM (actually ARM64 to be precise) does not have RDRAND/RDSEED but has RNDR - if I'm not mistaken.
If we become that specific, we should make sure to not tie it too much into x86_64 language.